### ISSUE TITLE
FEATURE: add player ability to set length of wordle word (2hr)

### ISSUE THREAD

**Issue by jymbocala** [jymbocala/wordle-terminal-app#21]
- [x] create options menu after player has inputed their name
  - [x] type 'start' to start game
  - [x] type 'options' to adjust game settings
- [x] add option to set length of wordle word 
- [x] add functionality to ask length of word they want (set to minimum 3 or maximum 7)
- [x] set wordle word length according to user's choice
- [x] adjust wordle classes with wordle word length
- [x] print "the word length has been updated"
- [x] ask if user is ready
  - [x] type 'start' to start game
  - [x] type 'length<number>' to re-adjust wordle word length
- [x] test

**Comment by jymbocala** [comment]
error: setting the word length in the options menu does not carry over to when the player starts the game.

**Comment by jymbocala** [comment]
I was able to refactor the overall structure of my code to not only fix the error above but to help with the separation of concern. Furthermore, I would be able to add more settings in the future easier.

1. I created an Options class in a separate file 
2. Created default settings
3. Added methods to adjust settings
4. Initiate the game in main.py using settings
game_round = wordle.GameRound(wordle_word, game, settings)
